lib/CL/devices/almaif/openasip/AlmaifCompileTCE.hh

```diff
@@ -32,6 +32,10 @@ int pocl_almaif_tce_initialize(cl_device_id device, const char *parameters);
 int pocl_almaif_tce_cleanup(cl_device_id device);
 void pocl_almaif_tce_compile(_cl_command_node *cmd, cl_kernel kernel,
                              cl_device_id device, int specialize);
+void pocl_almaif_tce_produce_standalone_program(AlmaifData* D, 
```

Comment:
did you run clang-format on these files?

Reply:
I thought I did, but apparenly format-last-commit.sh and format-branch.sh have a slightly different file regex, so these didn't get picked up by format-last-commit.sh.
format-branch.sh has:
'(.*(\.hh$|\.cc$))|(lib/llvmopencl/.*\.h)|(lib/CL/devices/tce/.*)'
And format-last-commit.sh:
'(.*(\.hpp$|\.cc$|\.cpp$))|(lib/llvmopencl/.*\.h)'
Which one should it be? And actually what would be the preferred naming convention for (these) C++ sources and headers?

Reply:
Never mind, you seem to have fixed these in the master in the meantime. Well, now after the rebasing to master, the format-branch.sh doesn't do anything anymore, so this should now be formatted correctly.

Reply:
I've added the .cpp extension to `format-last-commit.sh` few weeks ago, but i forgot to add it to the `format-branch.sh`. We have some .cpp files so it's needed; actually i think we also need both .hh and .hpp; not sure why is `lib/CL/devices/tce` there.